pulp_rpm/tests/functional/api/test_download_policies.py

```diff
@@ -1,110 +1,72 @@
 """Tests for Pulp`s download policies."""
-from pulp_smash.pulp3.bindings import PulpTestCase, monitor_task
-from pulp_smash.pulp3.utils import gen_repo, get_added_content_summary, get_content_summary
+import pytest
 
 from pulp_rpm.tests.functional.constants import (
     RPM_FIXTURE_SUMMARY,
     DOWNLOAD_POLICIES,
 )
-from pulp_rpm.tests.functional.utils import (
-    gen_rpm_client,
-    gen_rpm_remote,
-    skip_if,
-)
-from pulp_rpm.tests.functional.utils import set_up_module as setUpModule  # noqa:F401
-
-from pulpcore.client.pulp_rpm import (
-    RepositoriesRpmApi,
-    RpmRepositorySyncURL,
-    RemotesRpmApi,
-    RpmRpmPublication,
-    PublicationsRpmApi,
-)
-
-
-class SyncPublishDownloadPolicyTestCase(PulpTestCase):
-    """Sync a repository with different download policies.
-
-    This test targets the following issue:
-
-    `Pulp #4126 <https://pulp.plan.io/issues/4126>`_
-    `Pulp #4213 <https://pulp.plan.io/issues/4213>`_
-    `Pulp #4418 <https://pulp.plan.io/issues/4418>`_
+from pulpcore.client.pulp_rpm import RpmRpmPublication
+
+
+"""Sync a repository with different download policies.
+
+This test targets the following issue:
+
+`Pulp #4126 <https://pulp.plan.io/issues/4126>`_
+`Pulp #4213 <https://pulp.plan.io/issues/4213>`_
+`Pulp #4418 <https://pulp.plan.io/issues/4418>`_
```

Comment:
FWIW you can remove all of these going forwards.  Not all that useful anymore.  But no need to edit existing PRs.